backend/organization/views/project_views.py

```diff
@@ -846,6 +849,14 @@ def get_queryset(self):
         return ProjectStatus.objects.all()
 
 
+class ListProjectTypeOptions(ListAPIView):
+    permission_classes = [AllowAny]
```

Comment:
Why is the permission here `AllowAny`? Are we allowing another filter to the project page?

Reply:
I chose AllowAny because this is publically available data. In the future we will introduce a filter for this which will make AllowAny necessary. Would your preference be to change it to isAuthenticated now and return to AllowAny when we need it?